src/seb/registered_models/e5_models.py

```diff
@@ -26,12 +26,12 @@ def encode(
         self,
         sentences: list[str],
         *,
-        task: Task,  # noqa: ARG002
+        task: Task,
         batch_size: int = 32,
         **kwargs: Any,
     ) -> ArrayLike:
         sentences = self.preprocess(sentences)
-        return self.mdl.encode(sentences, batch_size=batch_size, **kwargs)  # type: ignore
+        return self.mdl.encode(sentences, batch_size=batch_size, task=task, **kwargs)  # type: ignore
```

Comment:
just a bug fix. Wasn't passed along.

Reply:
(we could add a test for each model type, e5 small + sentence transformer)

Reply:
+1 I think just seeing whether different models run without errors would be a great test, but then let's do it at the CLI level with maybe one task so that it's more of an integration test rather than a unit test